proposals/lip-0057.md

```diff
@@ -1945,38 +1989,17 @@ def afterTransactionsExecute(b: Block) -> None:
     # Notice that the keys in the substore naturally have the right ordering
     # when being read from the end to the beginning of the store.
 
-    roundNumber = roundNumber(height)
-
     snapshotState = {
             "validatorWeightSnapshot": eligibleValidators
         }
     create an entry in the snapshot substore with
-        storeKey = roundNumber.to_bytes(4,'big'),
+        storeKey = roundNumber(height).to_bytes(4,'big'),
```

Comment:
Did we align with the implementation here? Otherwise we could use this PR also to fix the misalignment.

Reply:
Yep, it is supposed to be fixed in the implementation by this [PR](https://github.com/LiskHQ/lisk-sdk/pull/8142/commits/72027e7cdee9d697df49f53cb994dfad844a0f15) which I will review today.